app/src/main/java/com/iic/shopingo/ui/request_flow/activities/CreateShoppingListActivity.java

```diff
@@ -49,17 +48,38 @@ protected void onCreate(Bundle savedInstanceState) {
       priceView.setText(Integer.toString(shoppingList.getOffer()));
     }
 
-    currencyView.setText(Currency.getInstance(Locale.getDefault()).getSymbol());
+    priceView.addTextChangedListener(this);
   }
 
   @OnClick(R.id.create_request_create_button)
   public void onCreateRequest(View view) {
     shoppingList.setItems(itemListView.getAllItems());
-    shoppingList.setOffer(Integer.parseInt(priceView.getText().toString()));
+    shoppingList.setOffer(Integer.parseInt(priceView.getText().toString().substring(1)));
     Intent intent = new Intent(this, SaveRequestActivity.class);
     intent.putExtra(SaveRequestActivity.EXTRAS_SHOPPER_KEY, shopper);
     intent.putExtra(SaveRequestActivity.EXTRAS_SHOPPING_LIST_KEY, shoppingList);
     startActivity(intent);
     finish();
   }
+
+  @Override
+  public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+  }
+
+  @Override
+  public void onTextChanged(CharSequence s, int start, int before, int count) {
+  }
+
+  @Override
+  public void afterTextChanged(Editable s) {
+    String symbol = Currency.getInstance(Locale.getDefault()).getSymbol();
+
+    String text = s.toString();
+    if (!text.contains("$")) {
```

Comment:
Also, we don't really handle different currencies in the server, so might as well just leave it as `$` for now.